weaver/tests/network-setups/corda/makefile

```diff
@@ -95,10 +103,19 @@ rebuild-network2:
 	cd dev/Corda_Network2/build/nodes/PartyB && rm -rf corda.jar djvm logs cordapps/*.jar
 	cd dev/Corda_Network2/build/nodes/PartyC && rm -rf corda.jar djvm logs cordapps/*.jar
 	rm -rf dev/Corda_Network2/build/nodes/.cache
+	rm -rf dev/Corda_Network2/build/kotlin
 	touch dev/Corda_Network2/build/nodes/Notary/cordapps/config/.gitkeep
 	touch dev/Corda_Network2/build/nodes/PartyA/cordapps/config/.gitkeep
 	touch dev/Corda_Network2/build/nodes/PartyB/cordapps/config/.gitkeep
 	touch dev/Corda_Network2/build/nodes/PartyC/cordapps/config/.gitkeep
+	mkdir -p dev/Corda_Network2/build/nodes/Notary/logs/
+	mkdir -p dev/Corda_Network2/build/nodes/PartyA/logs/
+	mkdir -p dev/Corda_Network2/build/nodes/PartyB/logs/
+	mkdir -p dev/Corda_Network2/build/nodes/PartyC/logs/
+	touch dev/Corda_Network2/build/nodes/Notary/logs/.gitkeep
+	touch dev/Corda_Network2/build/nodes/PartyA/logs/.gitkeep
+	touch dev/Corda_Network2/build/nodes/PartyB/logs/.gitkeep
+	touch dev/Corda_Network2/build/nodes/PartyC/logs/.gitkeep
```

Comment:
Why do we need to retain these logs?

Reply:
We just want to retain the empty directory, not the log files itself.